HOW_TO_RUN.md

```diff
@@ -1,52 +1,60 @@
-Thi
+# starbake
 
-## Install 
+## Prerequisites
+
+Before installing starbake, ensure the following minimum versions are installed on your system:
+
+- jdk: 11 or higher
+- python: 3.8 or higher
+
+## Install
 
 1. Install Starlake
-    `SL_VERSION=1.0.0-SNAPSHOT sh /Users/elarib/Work/Starlake/starlake/distrib/starlake.sh`
-2. Generate dummy files
-   `pip install faker`
-   `python _scripts/dummy_data_generator.py`
+`sh <(curl https://raw.githubusercontent.com/starlake-ai/starlake/master/distrib/setup.sh) --target=.`
+2. Create a virtual environment (optional)
+   `python3 -m pip install virtualenv`
+   `python3 -m venv .venv`
+3. Activate the virtual environment (optional)
+   `source .venv/bin/activate`
+4. Generate dummy files
+   `python3 -m pip install -r _scripts/requirements.txt`
+   `python3 _scripts/dummy_data_generator.py`
 
 We're good to go
 
-
-## Run Starlake 
+## Run Starlake
 
 1. Import from incoming to pending
-`sh starbake.sh import`
-
+`./starlake import`
 
 2. Load data to bigquery
```

Comment:
Should we change this to Load Data as parquet ? Because this (and based on the metadata/application.sl.yml) it does load data to local dir as parquet.
![image](https://github.com/starlake-ai/StarBake/assets/6231666/e833f3f6-8ab9-4476-b1c7-30e95a93f3ec)

Reply:
took into account